scvi/models/vae.py

```diff
@@ -292,6 +292,7 @@ def inference(
         """
         x_ = x
         if self.log_variational:
+            # + doesnt operate with float16
```

Comment:
don't get  this comment?

Reply:
this comment was just for me when testing. basically the '+' operator doesn't run with the variables on either side are torch.float16 interestingly. Removed the comment since it's pretty irrelevant.